tests/test_deprecations.py

```diff
@@ -196,3 +197,76 @@ def test_deprecations_find_nearest_sph():
     if version.parse(pf.__version__) >= version.parse('0.8.0'):
         with pytest.raises(TypeError):
             coords.find_nearest_sph(1, 1, 1, 1)
+
+
+def test_deprecations_freq_range_parameter_warnings():
```

Comment:
Can you add a test for each function that will check if everything was corrsctly deprecated, e.g., like this one (not sure which error is raised, though):
```python
if version.parse(pf.__version__) >= version.parse('0.8.0'):
        with pytest.raises(TypeError):
            coords.find_nearest_k(1, 0, 0)
```

Reply:
Oh, sure - totally forgot that. Should be a `TypeError` too, right?
Must be the same kind of error as in this:

```
def function(arg1, arg2, new_arg):
     return arg1, arg2, new_arg

function(1, 2, old_arg=3)

---------------------------------------------------------------------------
TypeError: function() got an unexpected keyword argument 'old_arg'
```